src/libPersistence/Retriever.cpp

```diff
@@ -138,9 +145,17 @@ bool Retriever::RetrieveTxBlocks(bool trimIncompletedBlocks) {
             }
           }
           // commit the state to disk
-          AccountStore::GetInstance().MoveUpdatesToDisk();
+          if (!AccountStore::GetInstance().MoveUpdatesToDisk()) {
+            LOG_GENERAL(WARNING, "AccountStore::MoveUpdatesToDisk failed");
+            return false;
+            ;
+          }
           // clear the stateDelta db
-          BlockStorage::GetBlockStorage().ResetDB(BlockStorage::STATE_DELTA);
+          if (!BlockStorage::GetBlockStorage().ResetDB(
+                  BlockStorage::STATE_DELTA)) {
+            LOG_GENERAL(WARNING, "BlockStorage::ResetDB (STATE_DELTA) failed");
```

Comment:
How about move the error message into `ResetDB` then we can handle the debug message in one place for easier maintenance. For caller here, it would be simpler to check the function result only.